frontend/src/views/CanteenEditor/SatelliteManagement.vue

```diff
@@ -159,7 +161,7 @@ export default {
       this.$store
         .dispatch("addSatellite", { id: this.canteen.id, payload: this.satellite })
         .then((satellite) => {
-          this.existingSatellites.push(satellite)
+          this.existingSatellites.unshift(satellite)
```

Comment:
If the cantine is a duplicate it gets added to the table again (even if with a refresh it goes away as it should). This could potentially be confusing because it looks successful at first but nothing has changed in the back. How about a 404 if the siret matches an existing satellite ?

Reply:
J'ai ajouté une étape de fetch après l'ajout d'un satellite : https://github.com/betagouv/ma-cantine/pull/1579/commits/774e48d50c5c9a9527025d7c6888b7cdf7c38067